biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceBuilder.java

```diff
@@ -884,6 +911,10 @@ public void copyCapabilities(Set<String> ignoreNamespaces, Resource r) {
 
 			addCapability(c);
 		}
+		if (r instanceof SupportingResource sr) {
```

Comment:
No the original idea was that a resource could need some crutches, not that they would be variations on the same bundle. Now the idea that we have a more general concept that you need that can support different models is not a bad idea, and that would require a name & API semantics change. We probably then need to delegate the expansion to the Supporting Resource. Need to think about it, open for suggestions.

Main concern here is that you need to know the semantics a capability copy operation? That falls very far from my intended meaning and I am actually not sure it is even needed for your use case. A SupportingResource == a resource and if you copy its capabilities you should not copy its members. The whole idea that we have this thing is because we cannot describe it as a single resource, this kills that idea.